pulse-service.http

```diff
@@ -0,0 +1,46 @@
+
+### Send Pulse backfill POST ALL request
+POST http://localhost:8612//api/pulse/metric/backfill
```

Comment:
not sure if the double slash is needed... guessing no, and probably just a noop.